src/components/ui/HamburgerMenu.tsx

```diff
@@ -53,31 +21,30 @@ const HamburgerMenu = () => {
           <div></div>
         </label>
       </Tooltip>
-      <ul className={`menuNav ${hamburgerOpen ? " showMenu" : ""}`}>
-        <li
-          className="listitem licolor1"
-          onClick={clickRules}
-          style={{
-            marginTop: "120px",
-            color: "white",
-          }}
-        >
-          <ListAltIcon style={{ width: "20%", paddingRight: "4px" }} />
-          The Rules
-        </li>
-        <Grow in={showRules}>{theRules}</Grow>
-        <li
-          onClick={clickLeave}
-          className="listitem licolor2"
-          style={{
-            marginTop: "8%",
-            color: "white",
-          }}
-        >
-          <SendIcon style={{ width: "20%", paddingRight: "4px" }} />
-          Leave Game
-        </li>
-      </ul>
+      <Slide direction="right" in={hamburgerOpen} mountOnEnter unmountOnExit>
+        <ul className="menuNav showMenu">
+          <RulesButton
+            style={{
+              marginTop: "50%",
+              color: "white",
+              left: "5%",
+              width: "80%",
+            }}
+          />
+          <LeaveButton
+            style={{
+              marginTop: "10%",
+              color: "white",
+              left: "5%",
+              width: "80%",
+            }}
+          />
+        </ul>
+      </Slide>
+      <p className={"hamburgerinfotext"}>
```

Comment:
This overlaps with other texts.

![grafik](https://user-images.githubusercontent.com/1506818/169690608-4b60064f-6b38-4861-9fcb-8c92e1ae3157.png)

Maybe you could use a tooltip for that, where you don't use the hover trigger, but control it yourself?

Reply:
I fixed it by creating a second Tool tip that disappears after 5 seconds - im not sure how good the quality of this code is so please review the changes in HamburgerMenu.tsx when i push this next commit